inst/rmarkdown/templates/measles_outbreak/skeleton/skeleton.Rmd

```diff
@@ -1116,27 +1120,30 @@ cowplot::plot_grid(
 ```
 
 
-We could then also look at admissions and exits by epiweek for operational purposes. 
+We could then also look at admissions and exits among inpatients by epiweek for operational purposes. 
```

Comment:
actually what we should do here is group by inpatient/outpatient and show admissions that way. Then show exits by inpatients only.